MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -19,19 +21,28 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		refresh()
 	}
 
-	@IBAction private func refresh() {
+	@IBAction func refresh() {
```

Comment:
There's no need to call `refresh` outside of `FeedViewController`, so this function should be marked as private.
```suggestion
	@IBAction private func refresh() {
```